providers/fab/src/airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -151,6 +151,19 @@ def init(self) -> None:
         if self.appbuilder:
             self._sync_appbuilder_roles()
 
+    @property
+    def is_airflow_2(self) -> bool:
+        """
+        Return whether the instance is run in Airflow 2 or Airflow 3 web application.
+
+        Can be deleted once the Airflow 2 legacy UI is removed.
+        """
+        return getattr(self, "airflow_2", None) is True
```

Comment:
```suggestion
        is_in_fab: bool = False
        """
        Whether the instance is run in FAB or React.
        Can be deleted once the Airflow 2 legacy UI is removed.
        """
```

couldn't we just do this instead? (might need to move it up a bit, cant suggest up there though)

Reply:
Simpler indeed :)